arlo-client/src/components/Audit/RoundManagement/RoundProgress.tsx

```diff
@@ -1,8 +1,60 @@
 import React from 'react'
+import { ProgressBar } from '@blueprintjs/core'
+import styled from 'styled-components'
 import H2Title from '../../Atoms/H2Title'
+import { IBallot, IAuditBoard } from '../../../types'
+import { IRound } from '../useRoundsJurisdictionAdmin'
 
-const RoundProgress = () => {
-  return <H2Title>Round 1 Progress</H2Title>
+const MainBarWrapper = styled.div`
+  margin-bottom: 20px;
+  width: 500px;
+  font-size: 16px;
+`
+
+const SmallBarWrapper = styled.div`
+  margin-bottom: 10px;
+  width: 400px;
+`
+
+const RoundProgress = ({
+  rounds,
+  ballots,
+  auditBoards,
+}: {
+  rounds: IRound[]
+  ballots: IBallot[]
+  auditBoards: IAuditBoard[]
+}) => {
+  const completeBallots = ballots.filter(b => b.status).length
```

Comment:
`b.status` will always be defined. It will be either `'AUDITED'`, `'NOT_AUDITED'`, or `'NOT_FOUND'`. So in this case, I think you want `ballots.filter(b => b.status !== BallotStatus.NOT_AUDITED)` (assuming you make an enum)

Reply:
@jonahkagan The enum is already there, so that works. :grin: